examples/mobile/UIComponents/js/app.js

```diff
@@ -36,12 +38,21 @@
 		dropDownMenuWidget.open();
 	}
 
-	document.addEventListener("pagebeforeshow", function () {
+	document.addEventListener("pageshow", function () {
+		themeChanger = document.querySelector(".ui-page-active select[data-native-menu='false']"),
+		themeChangerButton = document.querySelector(".ui-page-active .ui-btn-icon-more");
+		currentTheme = tau.theme.getTheme();
+
+		themeChanger.value = currentTheme;
```

Comment:
You didn't take account that themeChanger not exists.

Reply:
done, also themeChangerButton taken into account